### PR TITLE
Onboard Ignite to VPC Manager and Route Manager

### PR DESCRIPTION
**Pull request by chenpiaoping**
This PR includes the following features:
* Replace the db used by vpc_manager with ignite.
* Replace the db used by route_manager with ignite.
* Move the repo directory under the lib directory to the db directory.

### CODE REVIEW

File: services/route_manager/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+#spring.redis.host=10.103.179.92
+#spring.redis.port=6380
 #apache.kafka.address=172.17.0.1:9092
 #Logging configuration
 logging.level.root=info
 logging.level.org.springframework.web=info
 logging.file.path=.
 logging.type=file
+#Ignite configuration
+#ignite.host=192.168.131.131
```
Comment (xieus) [inline services/route_manager/src/main/resources/application.properties:13]: Is this change tested in K8s? I think we need to add ymal files in alcor/kubernetes/services folder.

Comment (kevin-zhonghao) [reply services/route_manager/src/main/resources/application.properties:13]: Yeah we need to add new ignite ymal files. I haven't test ignite in kubernetes before.

Comment (chenpiaoping) [reply services/route_manager/src/main/resources/application.properties:13]: Yes, I tested it when I deployed ignite in K8s.

Comment (chenpiaoping) [reply services/route_manager/src/main/resources/application.properties:13]: I have added some yaml files to ignite and tested the ignite deployment in K8s before.

Comment (chenpiaoping) [reply services/route_manager/src/main/resources/application.properties:13]: Here are some yaml files for ignite deployment in k8s: https://github.com/futurewei-cloud/alcor/tree/master/kubernetes/db/ignite

**Pull request merged**